.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build
+on:
+  create:
+    tags:
+       - 'v*.*'
+    
+jobs:
+  docker_build_push:
+    env:
+      GIT_REF: ${{ github.ref }}
+    if: startsWith(github.ref, 'refs/tags/v')
+
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Scala
+      uses: actions/setup-java@v2
+      with:
+        distribution: adopt
+        java-version: 11
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - name: Show output of sed to build.sbt command
+      run: |
+        VERS_TAG=$(echo $GIT_REF | sed 's/refs\/tags\/v//g')
+        VERS_TAG="\"$VERS_TAG"\"
+        sed "s/version := *.*/version := $VERS_TAG/" build.sbt
```

Comment:
It'd be nice if this were invariant to whitespace. I can never remember what makes sed happy but I'm thinking something like `s/version\s*:=\s*[0-9]+\.[0-9]+\.[0-9]+//` - just incase someone messes with the whitespace in the middle there.

Reply:
Implemented as the GitHub workflow trigger - reducing build hours spent on other steps - invalid version tag syntax will not trigger a build. If necessary can move this to sed regular expressions too.

Working with the correct supplied syntax: 
- https://github.com/GSS-Cogs/csvw-check/actions/runs/1545345235
- https://hub.docker.com/r/gsscogs/csvw-check/tags